charts/index.yaml

```diff
@@ -45,17 +45,17 @@ entries:
   cromwell:
   - apiVersion: v2
     appVersion: 68-8e12ab5
-    created: "2021-11-02T17:06:15.214495-04:00"
+    created: "2021-11-05T15:44:19.592712-04:00"
     description: Simple Cromwell deployment with Postgres database running jobs locally
-    digest: ae59ae9fa9765891d8e15f1e477901a55cbd1346621d7b7029298323c5f5fbf5
+    digest: c64a31d9c0720cdab89ba2e61f691fbea058994bfded94e1d5fa0387a5388be1
```

Comment:
Apologies if this was already discussed and decided but It feels not quite right to me that we're "changing history" by updating v0.1.6 rather than publishing a new 0.1.7 version with the new cromwell jar.

Reply:
@cahrens, I also had questions about why we are "overwriting" v.0.1.6 rather than creating a v.0.1.7. Is there a specific reason?

Reply:
Because @salonishah11 temporarily checked in 0.1.6 with the snapshot Cromwell version, and we planned to just adjust it to the actual Cromwell release once that was done. If you create 0.1.7 you will also need to update Leo, and that seems unnecessarily complex to me.

Reply:
Ah, I forgot about the SNAP version. This makes sense to me now. Thank you.